.github/actions/remove-mock-release-and-tag/action.yml

```diff
@@ -0,0 +1,57 @@
+name: Mock release cleanup
+description: Removal of release and tag
+inputs:
+  version:
+    description: 'Mock release version to be removed from tags and releases'
+    required: true
+  repo:
+    description: 'Path to repo'
+    required: true
+  gh-token:
+    description: 'gh token'
+    required: true
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      with:
+        repository: ${{ inputs.repo }}
+        token: ${{ inputs.gh-token }}
+
+    - uses: Alfresco/alfresco-build-tools/.github/actions/git-check-existing-tag@34b2301fc6245fa98ed73a82e04d4f9ab4eb5a3f #v3.6.0
+      id: check-tag
+      with:
+        tag: ${{ inputs.version }}
+
+    - name: Delete tag if it exist and ends with "-mock"
+      if: steps.check-tag.outputs.exists == 'true'
+      shell: bash
+      env:
+        GITHUB_TOKEN: ${{ inputs.gh-token }}
+        VERSION: ${{ inputs.version }}
+      run: |
+        if [[ "${VERSION}" == *-mock ]]; then
+          echo "Removing '${VERSION}' tag from: ${REPO}"
+          git push origin --delete ${VERSION}
+        else
+          echo "version does not end with '-mock'."
+          echo "Error: The provided version does not end with '-mock.'"
```

Comment:
Not sure why we need both echos? Last one seems enough (same below)

Reply:
IMO, only the second one is enough.

Reply:
first removed

Reply:
I still see it there

Reply:
fixed